src/StackShared/FrameStackNavigator.cs

```diff
@@ -291,7 +291,7 @@ async Task UIBack()
 
 					var result = await WaitForNavigationResult(() => _frame.GoBack(overrideInfo));
 
-#if NETFX_CORE
+#if WINDOWS_UWP
```

Comment:
Only on UWP or should this also be WinUI too?

Reply:
You're right, adding it